test/entity.ts

```diff
@@ -167,15 +264,17 @@ describe('entity', () => {
         namespace: 'namespace',
         path: ['ParentKind', 'name', 'Kind', 1, 'SubKind', new entity.Int('1')],
       });
+      const num = new entity.Int(1).valueOf();
```

Comment:
`num` and `str` are not used (perhaps you meant to replace the items in the `path` array below?